base/src/main/java/io/fabric8/launcher/base/http/HttpClient.java

```diff
@@ -48,6 +48,28 @@ public static HttpClient create() {
         return create(null);
     }
 
+    /**
+     * Utility function reading content of the body
+     * @param body Response stream from the server
+     * @return "null" so that it can be understood by Json parser as null node
+     * @throws IOException
+     */
+    public static String getContent(ResponseBody body) throws IOException {
+        if (body == null) {
+            return "null";
+        }
+        final String content;
+        final String bodyAsString = body.string();
+        if (bodyAsString == null || bodyAsString.trim().isEmpty()) {
```

Comment:
minor-- if we assign content like so: final String content = "nul" then we can negate the if statement,  assign content to bodyAsString in the if clause and eliminate the else clause

Reply:
That potentially makes it more concise. The benefit of the current approach is that having `String content` as `final` guarantees that you can't change it later on, only in `if-else` block. Having said that I reworked this method based on your suggestion. Shipped in 8b79d053